traitsui/qt4/enum_editor.py

```diff
@@ -139,7 +139,7 @@ def _get_inverse_mapping(self):
 
     # Trait change handlers --------------------------------------------------
 
-    def _values_changed(self, event=None):
+    def _values_updated(self, event=None):
```

Comment:
naming of methods here is ugly - there was a `values_changed` and `_values_changed` method, neither of which was a magically named listener since `values` is a trait on the factory, not on this object.  
I've made this update here, but we likely want to do some more renaming.  I'm happy to add that here or in a follow up PR - whichever a reviewer prefers.

Reply:
im not sure why we need to set `event=None` here

```suggestion
    def _update_values_and_rebuild_editor(self, event=None):
```

Reply:
ah you are right it is only `values_changed` that is called stand alone